.circleci/config.yml

```diff
@@ -48,21 +45,11 @@ jobs:
     steps:
       - *restore_repo
       - checkout
-      - restore_cache:
-          key: *base_config_key
-      - run:
-          name: 'Base environment setup'
-          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
-      - save_cache:
-          key: *base_config_key
-          paths:
-            - ~/.npmrc
-            - ~/.gitconfig
       - restore_cache:
           key: *yarn_cache_key
       - run:
-          name: Install Js dependencies
-          command: yarn install --no-progress --registry https://registry.verdaccio.org --no-lockfile
+          name: Install dependencies
```

Comment:
I would like to keep `--registry https://registry.verdaccio.org` otherwise will go to public in some scenarios.

Reply:
If we use `--registry` option, we have to remove `--frozen-lockfile`, so it will slow down the installation. Maybe we should have a .yarnrc that specifies the registry to install (like in monorepo)?